cmd/pbstatuser/main.go

```diff
@@ -73,6 +73,8 @@ func processMessage(ctx context.Context, message *kafka.GenericMessage) {
 		return
 	}
 
+	logger.Trace().Msgf("received a message from sources to be processed with source application id %s", s.SourceApplictionID)
```

Comment:
We don't have app id yet, but I guess we have source id, which should be enough? :)

Reply:
Yes sorry, it was a mistake. 
That is why I kept it on draft mode. 
Sorry